explorer/client/src/index.css

```diff
@@ -1,4 +1,9 @@
 @import 'vanilla-cookieconsent/dist/cookieconsent.css';
+
+@tailwind base;
```

Comment:
The effects of this PR on the UI end upon removing lines 3-5 here: 

```
@tailwind base;
@tailwind components;
@tailwind utilities;

```

Reply:
I ended up removing the CSS reset for now, and just ensuring that the components reset their styles fully for now. Once we've migrated some of the app to these components we should be able to remove the styles and re-introduce the CSS reset.